test/base-traits.jl

```diff
@@ -30,3 +30,8 @@ if VERSION < v"0.5.0-dev"
     # this give deprecation warning in Julia 0.5
     @test istrait(IsCallable{Base.AddFun})
 end
+
+if VERSION >= v"0.6.0-dev"
+    @test istrait(IsIterable{Array})
+    @test !istrait(IsIterable{Cmd})
```

Comment:
Let's add those two tests for good measure:
```
julia> method_exists(start, Tuple{Base.UnitRange{Int}})
true

julia> method_exists(start, Tuple{Base.UnitRange})
true
```
(I remember that parameterized methods sometimes tripped up `method_exists`.  So in case there is a regression.)